commit 577a81ec79ac1aa1cc8a45b122d2eb3c36051f78 (refs/changes/21/1121/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T22:16:41+00:00 (2 years, 10 months ago)
    
    Revert "TOOLS-1636 updates-imgadm et al with node v4 will not flush stdout on exit" (causes jenkins build failures)
    
    This reverts commit 37877d5c4eba6ec9c03678d2b4faaaa87011bb8d.

diff --git a/package.json b/package.json
index 69bd22e..f72a187 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "dependencies": {
     "manta": "3.1.3",
     "smartdc": "7.3.0",
-    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.3"
+    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
   },
   "engines": {
     "node": ">=0.8"
